src/native/tab_bar.rs

```diff
@@ -339,32 +349,61 @@ where
         }
     }
 
+    fn mouse_interaction(
+        &self,
+        layout: Layout<'_>,
+        cursor_position: Point,
+        _viewport: &Rectangle,
+        _renderer: &Renderer,
+    ) -> mouse::Interaction {
+        todo!()
+    }
+
     fn draw(
         &self,
         renderer: &mut Renderer,
-        defaults: &Renderer::Defaults,
+        _style: &iced_native::renderer::Style,
         layout: Layout<'_>,
         cursor_position: Point,
         viewport: &Rectangle,
-    ) -> Renderer::Output {
-        self::Renderer::draw(
-            renderer,
-            DrawEnvironment {
-                defaults,
+    ) {
+        let bounds = layout.bounds();
+        let children = layout.children();
+        let is_mouse_over = bounds.contains(cursor_position);
+        let style = if is_mouse_over {
```

Comment:
It's not necessary in this case here but I now prefer to use `let style_sheet = ...` to to better distinguish between the style of the renderer and the actual style sheet of the widget.